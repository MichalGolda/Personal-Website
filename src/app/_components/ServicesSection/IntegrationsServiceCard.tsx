import ServiceCard from "./ServiceCard";
import ServiceIcon from "./ServiceIcon";

const IntegrationServiceIcon = () => (
  <ServiceIcon src="/integration.webp" alt="integration" />
);

const IntegrationsServiceCard = () => (
  <ServiceCard
    title="Integracje z API i usługami analitycznymi"
    content={
      <>
        Integracje z zewnętrznymi API, umożliwiającymi rozszerzenie
        funkcjonalności strony internetowej poprzez połączenie z różnymi
        serwisami i platformami. <br />
        <br />
        Podpięcie narzędzi analitycznych, takich jak Google Analytics, Sentry i
        inne, aby dostarczyć klientom kompleksowe raportowanie i analizę danych
        dotyczących ruchu na stronie oraz wykrywanie ewentualnych problemów.
      </>
    }
    icon={<IntegrationServiceIcon />}
    data-aos="fade-left"
    data-aos-duration="2000"
  />
);

export default IntegrationsServiceCard;
