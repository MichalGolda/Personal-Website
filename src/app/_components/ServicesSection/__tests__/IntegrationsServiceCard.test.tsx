import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import IntegrationsServiceCard from "../IntegrationsServiceCard";

describe("IntegrationsServiceCard component", () => {
  it("renders unchanged", () => {
    const { container } = render(<IntegrationsServiceCard />);

    expect(container).toMatchSnapshot();
  });
});
