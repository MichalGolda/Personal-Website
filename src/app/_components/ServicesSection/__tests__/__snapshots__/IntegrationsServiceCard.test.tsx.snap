// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IntegrationsServiceCard component renders unchanged 1`] = `
<div>
  <div
    class="flex flex-col gap-y-4"
    data-aos="fade-left"
    data-aos-duration="2000"
  >
    <div
      class="flex flex-row gap-x-4 md:flex-col md:gap-y-2 lg:flex-row md:items-start lg:items-center items-center"
    >
      <img
        alt="integration"
        class="w-[48px] lg:w-[64px]"
        data-nimg="1"
        decoding="async"
        height="64"
        loading="lazy"
        src="/_next/image?url=%2Fintegration.webp&w=128&q=75"
        srcset="/_next/image?url=%2Fintegration.webp&w=64&q=75 1x, /_next/image?url=%2Fintegration.webp&w=128&q=75 2x"
        style="color: transparent;"
        width="64"
      />
      <h4
        class="font-bold text-lg md:text-xl lg:text-2xl text-secondary"
      >
        Integracje z API i usługami analitycznymi
      </h4>
    </div>
    <p
      class="text-body text-sm md:text-base"
    >
      Integracje z zewnętrznymi API, umożliwiającymi rozszerzenie funkcjonalności strony internetowej poprzez połączenie z różnymi serwisami i platformami. 
      <br />
      <br />
      Podpięcie narzędzi analitycznych, takich jak Google Analytics, Sentry i inne, aby dostarczyć klientom kompleksowe raportowanie i analizę danych dotyczących ruchu na stronie oraz wykrywanie ewentualnych problemów.
    </p>
    <span
      class="block bg-primary w-32 h-0.5"
    />
  </div>
</div>
`;
